src/components/ContentPill.tsx

```diff
@@ -141,7 +138,7 @@ const ContentPill = ({
     const length = flatListData?.length || 0
     return {
       length,
-      offset: length * index,
+      offset: (index - 1) * ITEM_SIZE,
```

Comment:
@tyler-whitman I'm not sure what the desired effect here is, but there was a bug when clicking on indexes beyond what could fit on the screen.

Reply:
Ah ok, this looks good. It was just to make sure the item selected is in view when you choose a hotspot from the list that is further down in the pill than the first page. Its working well for me still